integration/spark/vendor/gcp/src/main/java/io/openlineage/spark/agent/vendor/gcp/GcpVendor.java

```diff
@@ -0,0 +1,25 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.spark.agent.vendor.gcp;
+
+import io.openlineage.spark.agent.vendor.gcp.lifecycle.plan.GcpEventHandlerFactory;
+import io.openlineage.spark.agent.vendor.gcp.util.GCPUtils;
+import io.openlineage.spark.api.OpenLineageEventHandlerFactory;
+import io.openlineage.spark.api.Vendor;
+import java.util.Optional;
+
+public class GcpVendor implements Vendor {
+
+  @Override
+  public boolean isVendorAvailable() {
+    return GCPUtils.isDataprocRuntime();
```

Comment:
Wouldn't this clash later with other options to run Spark on GCP? Thinking loudly: wouldn't be better to have `GcpDataprocVendor` or `DataprocVendor`? I don't mind staying as is, just pointing out in case gcp spark runtime gets extended.

Reply:
make sense to rename it to GcpDataprocVendor